### DISCUSSION TITLE
Issues with partial Portuguese/Polish logos showing up…

### DISCUSSION THREAD

**Discussion by ps23Rick** [jellyfin/jellyfin#8794] [Q&A] [answered]
I’m running 10.8.7 of Jellyfin…

I just started noticing this today .. but don’t believe it was an issue before.  I see it directly on the Jellyfin website in addition to using with Infuse — both show the same issue.  If I force the language on a movie to English it has no effect.  I see this with some of my videos but not all.  Some are in Portuguese, some in polish, some other languages… 

Is there some setting I missed or perhaps which got altered?  Thoughts?  Below is one example..

![ED09E0F4-44D1-48E3-96EA-395CE27F8D63](https://user-images.githubusercontent.com/38017680/203908355-b02037b3-6b9c-4b79-9b96-02fa1671ba3f.jpeg)

**Comment by pypp** [comment]
You want to disable the movie logo ?

**Comment by ps23Rick** [comment]
Not necessarily — I like that but not sure why its sometimes in some other languages..

**Comment by pypp** [comment, marked as answer]
I think that when the logo fetching is done the first logo in the list is being downloaded, so that might not always be english logo. If you want you can change the logo manually.

**Reply by ps23Rick** [reply]
Got it.. I just popped in to check on that and there are no logos for this particular movie in english.. if I choose the “all languages” setting then I get two and they’re both in Portuguese.. I’m wondering if this could be considered a bug — choosing logos that are not the native language when initially selecting the meta data automatically.. ?

For me at least I know how to fix it so I’m good.. Thx!
